src/atoms/Link/Link.native.js

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { ThemeContext } from 'styled-components/native';
+import automation from '../../_helpers/automation-attributes';
+import Flex from '../Flex/Flex.native';
+import { TouchableOpacity } from 'react-native';
+
+const linkToTextSize = {
+  small: 'xsmall',
+  medium: 'medium',
+  large: 'large',
+};
+
+const Link = ({ children, visited, onClick, size, testID }) => {
+  const theme = useContext(ThemeContext);
+  const [active, setActive] = useState(false);
+  const inactiveTextColor = visited ? theme.colors.primary[800] : theme.colors.primary[800];
+  const activeTextColor = visited ? theme.colors.primary[600] : theme.colors.primary[600];
```

Comment:
I didn't understand this? Irrespective of the `visited` prop we are rendering `theme.colors.primary[600]` 🤔

Reply:
Because the design was changed to not have a different color for visited links but as per our discussion in standup, we wanted to leave the code untouched so that it can be accomodated later

Reply:
Removed visited now